internal/common/baseclient/baseclient.go

```diff
@@ -126,18 +125,9 @@ func (c *BaseClient) Logf(format string, a ...interface{}) {
 // OPTIONAL. Base should be able to handle it but feel free to implement your own.
 func (c *BaseClient) StartOfTurnUpdate(gameState gamestate.ClientGameState) {
```

Comment:
Do we even need `StartOfTurnUpdate` any more?

Reply:
Yusuf said we should leave it. People might want to log stuff or do some kind of bookkeeping there.